TAO/TAO_IDL/util/utl_err.cpp

```diff
@@ -185,9 +185,11 @@ error_string (UTL_Error::ErrorCode c)
     case UTL_Error::EIDL_KEYWORD_WARNING:
       return "Warning - spelling differs from IDL keyword only in case: ";
     case UTL_Error::EIDL_ANONYMOUS_ERROR:
-      return "Error: anonymous types are deprecated by OMG spec";
+      return "Error: anonymous types are deprecated by OMG spec in IDL3, "
```

Comment:
Shouldn't these messages be updated to just say that anonymous types are disabled without referring to deprecated/IDL4 now that it is up to the user/backend to decide when they are not allowed or just trigger a warning?

Reply:
By default it is still based on the IDL version, so I'd like to keep that message when that is the case since it explicitly says what the problem is. I can do this by switching the error message based on if the setting has been explicitly set. For the warning message I can make it generic unconditionally.